src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -512,4 +500,28 @@ public static boolean isName(String str) {
 
         return isName;
     }
+
+    /**
+     * Returns true if the value arg is either null, or empty
+     * or contains only whitespace characters.
+     * More efficient that calling (string).trim().length() == 0
+     * or (string).trim().isEmpty()
+     * because it does not create temporary string whereas trim() does.
+     * This method is the same as Apache Сommons StringUtils.isBlank()
+     * but a bit refactored
+     * @param value A string to check
+     * @return true if the value does not contain any character except whitespace.
```

Comment:
@Kietzmann He is talking about the bottom line, 513. The bottom one is always the one we pick in the review.

@MEZk If we add everything it will be the same as main description, which to me seems repetitive and makes the Javadoc helpful in fully reading it.
What about rewriting the main description's first sentence to `Checks if the value arg blank by either being null, empty, or contains only whitespace characters.` and rewrite the return tag to `true if the arg is blank.`?